### PR TITLE
Feature/new user modal

### PR DESCRIPTION
**Pull request by mrlandino**
# Description drawing

We updated the login screen to reflect new inputs. The modal is built out with functionality to close when button is clicked and when you click outside of modal. All inputs and buttons on modal function with styling.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactor feature (non-breaking change that modifies existing work)
- [x] Cosmetic changes (changes to layout or visuals in CSS file)

# How Has This Been Tested?

- [ ] console.log()
- [x] dev tools
- [x] functionality check through web browser

# Checklist:

- [x] My code follows the style guidelines of this project (semicolons, indentation, naming conventions, etc..)
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have deleted all comments/console.log() after functionality and understanding is final
- [x] I have added any issues closed by this PR in the Issues section.
- [x] I have added others as reviewers

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -28,6 +28,7 @@
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <div id="root"></div>
```
Comment (userigorgithub) [inline public/index.html:30]: Great job using portal!

File: src/Components/login/Login.tsx [hunk 2]
```diff
@@ -6,16 +6,24 @@
 import { Navigate } from 'react-router-dom';
```
Comment (userigorgithub) [inline src/Components/login/Login.tsx:6]: Good job updating Login page according to BE api.

File: src/Components/modal/Modal.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
+import React, { useRef, useState } from 'react'
```
Comment (userigorgithub) [inline src/Components/modal/Modal.tsx:1]: GJ setting up Modal component for sign up registration!

File: src/Components/modal/Modal.tsx [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+import ReactDom from 'react-dom'
+import './Modal.scss'
+
+const Modal = ({ setShowModal }) => {
+  const [username, setUsername] = useState('')
+  const [email, setEmail] = useState('')
+  const modalRef = useRef();
+  const closeModal = (event) => {
```
Comment (userigorgithub) [inline src/Components/modal/Modal.tsx:9]: Nice use of closing modal if clicking outside of it.

**Review by userigorgithub — APPROVED** [review]
Great job on doing Modal Ross, and updating Login page. Amazing scss!!!

**Review by michaelputnam67 — APPROVED** [review]
Looks great!

**Pull request merged**